k8s/examples/wordpress/wordpress/10-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: wordpress
+  labels:
+    app: wordpress
+    tier: frontend
+spec:
+  externalTrafficPolicy: Local
+  ports:
+    - port: 8080
+      targetPort: 80
+      name: wordpress
+      protocol: TCP
+      name: http
+  selector:
+    app: wordpress
+    tier: frontend
+  type: NodePort # We use NodePort by default, but can use LoadBalancer
```

Comment:
We should use ClusterIP instead of a node port. The implications of using NodePort are significant. If you want to export that to the world without creating a LoadBalancer, I'd suggest to use an ingress rule (I can help on providing the components and specs for it)